.github/workflows/lint.yml

```diff
@@ -15,8 +15,8 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.18
+          go-version: 1.19
```

Comment:
we're still using go 1.18, does changing the version here just apply to the version of go that executes the linter?

Reply:
That's correct